templates/topic/index.json-ld

```diff
@@ -28,19 +29,195 @@
+				"fr": "Cette implémentation est destinée aux développeurs/éditeurs qui ajoutent le gabarit manuellement."
+			},
+			"instructions": {
+				"en": [
+					"Refer to the working example for a code sample.",
+					"Refer to the working example and guidance for more information on how to implement this page template."
+				],
+				"fr": [
+					"Reportez-vous à l'exemple pratique pour un exemple de code.",
+					"Référez-vous à l'exemple pratique et aux directives pour plus d'informations sur l'implémentation ce modèle de page."
+				]
+			}
+		},
+		{
+			"@id": "_:implement_topic_aem",
+			"iteration": "_:iteration_topic_3",
+			"name": {
+				"en": "AEM users",
+				"fr": "Utilisateurs AEM"
+			},
+			"introduction": {
+				"en": "This implementation is meant for publishers adding the template manually into an AEM page.",
+				"fr": "Cette implémentation est destinée aux éditeurs qui ajoutent le gabarit manuellement à une page d'AEM."
+			},
+			"instructions": {
+				"en": [
+					"TODO"
+				],
+				"fr": [
+					"TODO"
+				]
+			},
+			"ajaxSourceCode": {
+				"en": "topic-en.html main > *:not(.pagedetails)",
+				"fr": "topic-fr.html main > *:not(.pagedetails)"
+			}
+		}
+	],
+	"iteration": [
+		{
+			"@id": "_:iteration_topic_3",
+			"name": "Topic page - Iteration 3",
+			"date": "2024-06",
+			"breaking": [
+				"Stabilization of the topic page template."
+			],
+			"detectableBy": ".page-type-topic"
+		},
+		{
+			"@id": "_:iteration_topic_2",
+			"name": "Theme / Topic page (Beta) - Iteration 2",
+			"date": "2021-01",
+			"detectableBy": "Half-width image introduction block. Full width most requested (provisional). Social media channels at bottom right",
+			"example": {
+				"en": [
+					{
+						"href": "demoted/theme-topic-en.html",
+						"text": "Theme - Topic page version 2 (beta)"
+					}
+				],
+				"fr": [
+					{
+						"href": "demoted/theme-topic-fr.html",
+						"text": "Page the thème ou de sujet version 2 (bêta)"
+					}
+				]
+			}
+		},
+		{
+			"@id": "_:iteration_topic_1",
+			"name": "Topic page - Iteration 1",
+			"date": "2014-08",
+			"detectableBy": "Social media channels at top left. Most requested, Contributors and More information on the right.",
+			"example": {
+				"en": [
+					{
+						"href": "demoted/topic-en.html",
+						"text": "Topic page version 1"
+					}
+				],
+				"fr": [
+					{
+						"href": "deprecated/topic-fr.html",
+						"text": "Page de sujet version 1"
+					}
+				]
+			}
+		}
+	],
+	"changesets": [
+		{
+			"@id": "_:cs_topic_2",
+			"name": "Topic page",
+			"status": "stable",
+			"baseOnIteration": "_:iteration_topic_2",
+			"detectableBy": ".page-type-topic",
+			"layout": [
+				"Intro block",
+				"Most requested - version 1",
+				"Services and information - version 3",
+				"Context-specific features - version 5.0 [.col-md-8] + Social media channels - version 1 [.col-md-4]",
```

Comment:
"Social media" is not the "follow us" component? do we have 2 distinct component? I think that should be noted as being the version 2.

For the "features" see my note to confirm or not if we are using the co-habit pattern.

Reply:
You are right, it should be version 2.1.